jazz-ui/src/app/primary-components/report-issue/report-issue.component.scss

```diff
@@ -0,0 +1,176 @@
+  height: 34px;
+  text-align: center;
+  text-decoration: none solid rgb(51, 51, 51);
+  text-size-adjust: 100%;
+  width: 1340.4px;
+  column-rule-color: rgb(51, 51, 51);
+  perspective-origin: 670.2px 17px;
+  transform-origin: 670.2px 17px;
+  caret-color: rgb(51, 51, 51);
+  border: 0px none rgb(51, 51, 51);
+  font: normal normal 700 normal 24px / 34.2857px Lato;
+  outline: rgb(51, 51, 51) none 0px;
+}
+
+.icon-icon-server-error {
+  font-size: 7rem;
+}
+
+.report-issue-message {
+  box-sizing: border-box;
+  color: rgb(51, 51, 51);
+  height: 51.2px;
+  text-align: center;
+  text-decoration: none solid rgb(51, 51, 51);
+  text-size-adjust: 100%;
+  width: 1340.4px;
+  column-rule-color: rgb(51, 51, 51);
+  perspective-origin: 670.2px 25.6px;
+  transform-origin: 670.2px 25.6px;
+  caret-color: rgb(51, 51, 51);
+  border: 0px none rgb(51, 51, 51);
+  font: normal normal 500 normal 18px / 25.7143px Lato;
+  outline: rgb(51, 51, 51) none 0px;
+}
+
+.report-issue-submessage {
+  box-sizing: border-box;
+  color: rgb(51, 51, 51);
+  height: 22px;
+  text-align: center;
+  text-decoration: none solid rgb(51, 51, 51);
+  text-size-adjust: 100%;
+  width: 1340.4px;
+  column-rule-color: rgb(51, 51, 51);
+  perspective-origin: 670.2px 11px;
+  transform-origin: 670.2px 11px;
+  caret-color: rgb(51, 51, 51);
+  border: 0px none rgb(51, 51, 51);
+  font: normal normal 500 normal 15.6px / 22.2857px Lato;
+  outline: rgb(51, 51, 51) none 0px;
+}
+
+.text-pink {
+  color: rgb(237, 2, 149);
+  cursor: pointer;
+}
+
+.feedback-overlay{
+  position: fixed;
+  top:0;
+  right: 0;
+  width:100%;
+  height:100%;
+  background-color: rgba(0, 0, 0, 0.3);
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  z-index: 1;
+}
+
+.feedback-modal{
+  position: relative;
+  min-width: 500px;
+  max-width: 600px;
+  height: calc(350rem/12);
+  width:45%;
+  align-self: center;
+  background-color: #fff;
+  padding:2rem;
+  font-size: 16px;
+  .feedback-hdr{
+    font-size: 2rem;
+    font-weight: bold;
+  }
+}
+
+.feedback-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.separator-row{
+  border-bottom: 1px solid #ccc;
+  margin-top: 2rem;
+}
+
+.feedback-text-wrap{
+  display: flex;
+  flex-direction: column;
+  p{
+    padding: 1.5rem 0;
+  }
+  textarea{
+    padding: 10px 10px;
+    width: 100% !important;
+    height: 100px !important;
+    outline: none;
+  }
+
+}
+.feedback-text-wrap.feedbackRes{
+  position: absolute;
+  padding: 0 3rem;
+  top: 46%;
+  left: 0;
+}
+.feedback-btn-wrap{
+  margin-top: 10px;
+  font-size:13px;
+  display: flex;
+  align-items: center;
+  a{
+    align-self: right;
+    margin:5px;
+    cursor: pointer;
+  }
+  .feedback-submit{
+    align-self: right;
+    /deep/ button.btn-primary{
+      line-height:3rem;
+    }
+  }
+}
+/deep/ button.btn-primary:focus, button.btn-primary:active{
+  background-color: #ed008c !important;
+  color: #fff !important;
+  border: 1px solid #ed008c !important;
```

Comment:
are you guys sticking to using !important?

Reply:
this is part of the original component styles